src/views/Game/GameDock.tsx

```diff
@@ -322,8 +324,7 @@ export function GameDock({
         <Dock>
             {(tournament_id || null) && (
                 <Link className="plain" to={`/tournament/${tournament_id}`}>
-                    <i className="fa fa-trophy" title={_("This is a tournament game")} />{" "}
-                    {_("Tournament")}
+                    <i className="fa fa-trophy" title={_(`${tournament_name}`)} /> {_("Tournament")}
```

Comment:
2 comments:

- Don't need to wrap the tournament title in `_()`.  That function is for translation, but it's unlikely to work on user-generated names.
- Need to handle the `undefined` case, perhaps fall back to the old thing (or `_("Tournament")` or no tooltip at all)

Reply:
Great, thanks for the info!